ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/services/JogSizeAction.java

```diff
@@ -95,7 +95,7 @@ public void actionPerformed(ActionEvent e) {
         } else if (operation != null) {
             switch (operation) {
                 case STEPXY_MULTIPLY:
-                    js.multiplyXYStepSize();
+                    getJogService().multiplyXYStepSize(); //AndyCXL
```

Comment:
Thanks, and sorry for the late reply!

I think that it is important with attribution but is against of using it as comments like this as they clutter down the code. The attribution will be visible through git, making the comment a bit redundant.